.github/workflows/build-images.yaml

```diff
@@ -114,8 +117,14 @@ jobs:
         id: add-latest-tag
         run: |
           echo "IMG_TAGS=latest ${{ env.IMG_TAGS }}" >> $GITHUB_ENV
-      - name: Generate Catalog Content
-        run: make catalog
+      - name: Run make catalog
+        if: ${{ github.ref_name != env.MAIN_BRANCH_NAME }}
+        run: make catalog REGISTRY=${{ env.IMG_REGISTRY_HOST }} ORG=${{ env.IMG_REGISTRY_ORG }} IMAGE_TAG=${{ github.ref_name }}
+      - name: Run make catalog (main)
+        if: ${{ github.ref_name == env.MAIN_BRANCH_NAME }}
+        run: make catalog REGISTRY=${{ env.IMG_REGISTRY_HOST }} ORG=${{ env.IMG_REGISTRY_ORG }} IMAGE_TAG=latest VERSION=0.0.0
+      - name: Git diff
```

Comment:
pretty much useless, the catalog manifests are not committed in git and they are ignored via the `.gitignore` file

Reply:
Yeah, this confuses me as well… @didierofrivia ?